draft-kuhn-quic-careful-resume-03.xml

```diff
@@ -613,9 +613,10 @@ keywords will be used for the search engine. -->
             <t>Rationale #2 - Solution #3 : This approach is the same as
             above, except that the sender provides an estimate of the saved_rtt
             and saved_bb parameters in a form that may be read by the receiver.
-            The information might not be encrypted, or the information might
-            be duplicated outside of the encrypted block. This allows a receiver
-            to read, but not modify, the saved_rtt and saved_bb parameters and
+            Using the security mechanisms provided in <xref target="I-D.kuhn-quic-bdpframe-extension"></xref>,
+            the sender can guarantee that the receiver did not change the parameters inside the frame.
+	    The receiver can 
```

Comment:
/guarentee/ could be /confirm/  or /verify/ ?

Reply:
Fixed in https://github.com/NicoKos/resume_connection/commit/54e7a2d7187b1c32be213d231371f467d79edebb